Tools/TestWebKitAPI/Tests/WTF/NativePromise.cpp

```diff
@@ -1855,7 +1855,7 @@ class PhotoProducer : public ThreadSafeRefCounted<PhotoProducer> {
             // Note that you can resolve a NativePromise on any threads. Unlike with a CompletionHandler it is not the responsibility of the producer to resolve the promise
             // on a particular thread.
             // The consumer specifies the thread on which it wants to be called back.
-            producer.resolve(std::make_pair<Vector<uint8_t>, String>({ image.data(), image.size() }, { mimeType.data(), static_cast<unsigned>(mimeType.size()) }));
+            producer.resolve(std::make_pair<Vector<uint8_t>, String>(std::span { image.data(), image.size() }, { mimeType.data(), static_cast<unsigned>(mimeType.size()) }));
```

Comment:
Surprising that we can’t make a `std::span` from a `std::vector` without separately getting `data` and `size`.

Reply:
It does work. I'll fix.